emscripten/1_download_library.sh

```diff
@@ -16,22 +16,26 @@ msg " [1] Preparing Emscripten SDK"
 nproc=$(nproc)
 export MAKEFLAGS="-j${nproc:-2}"
 
-download_and_extract https://s3.amazonaws.com/mozilla-games/emscripten/releases/emsdk-portable.tar.gz
+rm -rf emsdk-portable
+git_clone https://github.com/emscripten-core/emsdk.git emsdk-portable
 cd emsdk-portable
 
 # Fetch the latest registry of available tools.
-./emsdk update
+./emsdk update-tags
 
 # Download and install the latest SDK tools and set up the compiler configuration to point to it.
-./emsdk install sdk-incoming-64bit
-./emsdk activate sdk-incoming-64bit
+./emsdk install 1.39.5
+./emsdk activate 1.39.5
 
 # Set the current Emscripten path
 # The following line fails to run in jenkins jobs. Running inner called scripts instead works.
-#. ./emsdk_env.sh
 ./emsdk construct_env
 . ./emsdk_set_env.sh
 
+# Activating emsdk overwrites a config file in the home directory
+# by copying it to emsdk the local file is used
+cp ~/.emscripten .
```

Comment:
I do not like this. Need to check, if there is a better way.

Reply:
I found a better solution that involves creating of an empty file in the emsdk directory